Weather/Weather/Data/Source/Local/Database.swift

```diff
@@ -93,11 +90,22 @@ final class Database: NSObject {
         return [T]()
     }
     
-    func getLocations() -> [Location] {
+    func getLocationsGPS() -> [Location] {
         var locations = [Location]()
         fetch(LocationCoreData.self).forEach {
             let coordinate = Coordinate(lat: $0.latitude, lng: $0.longitude)
-            locations.append(Location(placeId: $0.placeId, coordinate: coordinate))
+            locations.append(Location($0.placeId, coordinate: coordinate))
+        }
+        return locations
+    }
+    
+    func getLocationsNoneGPS() -> [Location] {
```

Comment:
func này và func trên có thể gộp làm 1
```suggestion
    func getLocations(isNeedGPS: Bool) -> [Location] {
```